.gitignore

```diff
@@ -1,15 +1,10 @@
-# Defaults
-
 .coverage
 *.egg-info
 *.log
 *.mo
 *.py?
 *.swp
-
-
-# Dirs
-
+# dirs
```

Comment:
What exactly is the benefit for the migration to rename a comment?

Reply:
Copied the .gitignore from the default bobtemplates output, who knows. Maybe THEY fixed that.